cmd/sifnoded/genaccounts.go

```diff
@@ -459,3 +460,73 @@ func SetGenesisOracleAdminCmd(
 
 	return cmd
 }
+
```

Comment:
Why is this in genesis?

Reply:
after the feature deployed, the ceth paid by accout will go to the specific account. If it is not set, the transaction will be failed. Other method could be we upgrade the network, then send a transaction to set the account before accept and process any lock/burn transaction.

Reply:
it is similar with admin account case.